package.json

```diff
@@ -1,65 +1,101 @@
+    "estraverse": "^4.3.0",
+    "events": "^3.3.0",
+    "fast-deep-equal": "^3.1.3",
+    "fast-json-stable-stringify": "^2.1.0",
+    "fastest-levenshtein": "^1.0.16",
+    "find-up": "^4.1.0",
+    "function-bind": "^1.1.1",
+    "glob-to-regexp": "^0.4.1",
+    "graceful-fs": "^4.2.11",
+    "has": "^1.0.3",
+    "has-flag": "^4.0.0",
+    "import-local": "^3.1.0",
+    "interpret": "^3.1.1",
+    "is-core-module": "^2.12.0",
+    "is-plain-object": "^2.0.4",
+    "isexe": "^2.0.0",
+    "isobject": "^3.0.1",
+    "jest-worker": "^27.5.1",
+    "json-parse-even-better-errors": "^2.3.1",
+    "json-schema-traverse": "^0.4.1",
+    "kind-of": "^6.0.3",
+    "loadash": "^1.0.0",
+    "loader-runner": "^4.3.0",
+    "locate-path": "^5.0.0",
+    "merge-stream": "^2.0.0",
+    "mime-db": "^1.52.0",
+    "mime-types": "^2.1.35",
+    "neo-async": "^2.6.2",
+    "node-releases": "^2.0.10",
+    "p-limit": "^2.3.0",
+    "p-locate": "^4.1.0",
+    "p-try": "^2.2.0",
+    "path-exists": "^4.0.0",
+    "path-key": "^3.1.1",
+    "path-parse": "^1.0.7",
+    "picocolors": "^1.0.0",
+    "pkg-dir": "^4.2.0",
+    "punycode": "^2.3.0",
+    "randombytes": "^2.1.0",
+    "rechoir": "^0.8.0",
+    "resolve": "^1.22.2",
+    "resolve-cwd": "^3.0.0",
+    "resolve-from": "^5.0.0",
+    "safe-buffer": "^5.2.1",
+    "schema-utils": "^3.1.2",
+    "serialize-javascript": "^6.0.1",
+    "shallow-clone": "^3.0.1",
+    "shebang-command": "^2.0.0",
+    "shebang-regex": "^3.0.0",
+    "source-map": "^0.6.1",
+    "source-map-support": "^0.5.21",
+    "supports-color": "^8.1.1",
+    "supports-preserve-symlinks-flag": "^1.0.0",
+    "tapable": "^2.2.1",
+    "terser": "^5.17.2",
+    "terser-webpack-plugin": "^5.3.8",
+    "update-browserslist-db": "^1.0.11",
+    "uri-js": "^4.4.1",
+    "watchpack": "^2.4.0",
+    "webpack": "^5.82.0",
+    "webpack-cli": "^5.1.0",
+    "webpack-merge": "^5.8.0",
+    "webpack-sources": "^3.2.3",
+    "which": "^2.0.2",
+    "wildcard": "^2.0.1"
   },
+  "devDependencies": {},
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "jest --runInBand",
-    "startHeadlessOnTestPort": "BROWSER=none PORT=$npm_package_config_testPort yarn start",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "devDependencies": {
-    "@babel/core": "^7.18.6",
-    "@babel/preset-env": "^7.18.6",
-    "babel-jest": "^28.1.1",
-    "chromedriver": "^100.0.0",
-    "geckodriver": "^3.0.1",
-    "jest": "^27.5.1",
-    "jest-puppeteer": "^6.1.0",
-    "puppeteer": "^15.1.1",
-    "react-string-replace": "^1.0.0",
-    "start-server-and-test": "^1.14.0"
-  }
```

Comment:
why did you remove all devDependencies? What package manager did you use?

F.e. I wasn't able to run the tests since jest was deleted from devDependencies.